synchronization/trigger_test.go

```diff
@@ -80,15 +80,17 @@ func TestPeriodicalTriggerStopOnContextCancel(t *testing.T) {
 func TestPeriodicalTriggerStopWorksWhenContextIsCancelled(t *testing.T) {
 	with.Concurrency(t, func(parent context.Context, harness *with.ConcurrencyHarness) {
 		ctx, cancel := context.WithCancel(parent)
-		cancel() // send a cancelled context to reduce chances of trigger being called even once
-		x := 0
-		p := synchronization.NewPeriodicalTrigger(ctx, "a periodical trigger", time.Millisecond*2, harness.Logger, func() { x++ }, nil)
+		var stage int32 = 0
```

Comment:
either add a comment for the meaning of 0 and 1, or define consts. It's nitpicking but it aids in readability